macro-fullcalendar-api/src/main/java/org/xwiki/fullcalendar/internal/DefaultFullCalendarManager.java

```diff
@@ -61,15 +66,30 @@ public class DefaultFullCalendarManager implements FullCalendarManager
 {
     private static final String T_VALUE = "T";
 
+    private static final String ID_JSON_KEY = "id";
+
+    private static final String START_DATE_JSON_KEY = "start";
+
+    private static final String END_DATE_JSON_KEY = "end";
```

Comment:
We usually have a common prefix, not a common suffix. I mean ``JSON_KEY_*``. See for instance https://github.com/xwiki/xwiki-commons/blob/xwiki-commons-15.8/xwiki-commons-core/xwiki-commons-job/xwiki-commons-job-api/src/main/java/org/xwiki/job/Request.java#L40 , but there are plenty of such examples.